src/NServiceBus.Router/InterfaceConfiguration.cs

```diff
@@ -6,11 +6,22 @@
     using Transport;
     using Unicast.Subscriptions.MessageDrivenSubscriptions;
 
+    /// <summary>
+    /// Represents an interface configuration,
+    /// </summary>
+    public interface IInterfaceConfiguration
```

Comment:
Why do we need this one?

Reply:
It was mainly to reuse the extension methods in [MessageDrivenSubscriptionExtensions.cs](https://github.com/SzymonPobiega/NServiceBus.Router/blob/master/src/NServiceBus.Router/MessageDrivenSubscriptionExtensions.cs).

Didn't want to refactor that much, though I'm sure there are opportunities here and there (inheritance etc.).

I can remove `IInterfaceConfiguration` if you'd like, and add an additional set of extension methods for `SendOnlyInterface<T>`?

Reply:
https://github.com/SzymonPobiega/NServiceBus.Router/pull/24/commits/7fa912b65f38bba730234a60edc8f56eb2ce576d